src/mumble/main.cpp

```diff
@@ -496,54 +535,27 @@ int main(int argc, char **argv) {
 
 	Themes::apply();
 
-	QString qsSystemLocale = QLocale::system().name();
+	QLocale systemLocale = QLocale::system();
 
 #ifdef Q_OS_MAC
 	if (os_lang) {
-		qWarning("Using Mac OS X system language as locale name");
-		qsSystemLocale = QLatin1String(os_lang);
+		const QLocale macOSLocale = QLocale(os_lang);
+
+		if (macOSLocale != QLocale::c()) {
+			qWarning("Using Mac OS X system language as locale name");
+			systemLocale = macOSLocale;
+		}
 	}
 #endif
 
-	const QString locale = Global::get().s.qsLanguage.isEmpty() ? qsSystemLocale : Global::get().s.qsLanguage;
-	qWarning("Locale is \"%s\" (System: \"%s\")", qPrintable(locale), qPrintable(qsSystemLocale));
-
-	QTranslator translator;
-	if (translator.load(QLatin1String(":mumble_") + locale))
-		a.installTranslator(&translator);
+	QLocale settingsLocale = QLocale(Global::get().s.qsLanguage);
+	if (settingsLocale == QLocale::c()) {
+		settingsLocale = systemLocale;
+	}
```

Comment:
This means it looks for either the locale that is stored in the settings, or the current OS locale.

So if a translator wants to test a specific language they have to select and save this setting in the Mumble settings dialog.

However, the settingsdialog, I presume, will only displayed the languages that we embedded. Consequently, I assume, a translator would not be able to test a newly added translation (or once we only embed reasonably complete translations a more complete translation of what was left out)?

Reply:
If that is the case it is fine to me not to include it, but it is a noteworthy caveat and explicit exclusion of being tackled here.

Reply:
Probably. But this is exactly as it was before. How does that work right now?

Reply:
I could imagine adding another parameter though that allows for specifying a locale on the command line...

Reply:
Done